pydis_site/apps/api/views.py

```diff
@@ -305,7 +305,14 @@ def post(self, request: Request, *, webhook_id: str, webhook_token: str) -> Resp
         )
         headers.pop('Connection', None)
         headers.pop('Content-Length', None)
```

Comment:
We don't need to do this anymore, right?

Reply:
Good spot, removed in 346d570